src/posts/blog/index.ts

```diff
@@ -2,10 +2,12 @@ import uxredesign from 'posts/blog/1-ux-redesign/ux-redesign'
 import thanksgiving from 'posts/blog/2-thanksgiving/thanksgiving'
 import hepafilters from 'posts/blog/3-hepa-filters/hepa-filters'
 import negativetests from 'posts/blog/4-negative-tests/negativeTests'
+import b117 from 'posts/blog/5-b117/b117'
 import { BlogPostMap } from 'posts/post'
 
 // Note: the keys in this map are the public URL for the post. Changing them will break links.
 export const pages: BlogPostMap = {
+  b117,
```

Comment:
This is kind of terrible SEO. Not that it matters much. But could we do something more friendly to the average non-technical user. Like `newcovidstrain` or something?

Reply:
Is it? I expect folks to be searching B117 if they want to know about this one in particular. Also SEO should look at the page contents and title, which reference the new variant. Also "strain" is scientifically incorrect, SARS-CoV-2 is a strain, B117 is a variant. Also there are new variants all the time, this isn't the first or the last.

Reply:
Oh thanks for the strain vs variant FYI.

Okay I'm sold. Makes ense.